### PR TITLE
Update m68k-amigaos.cmake

### PR DESCRIPTION
**Pull request by MBeijer**

### CODE REVIEW

**Comment by simpsond** [comment]
It's funny you added this, I was just working on a library build last night and was realizing the way to unset this was awkward at best.

**Review by simpsond — APPROVED** [review]

File: m68k-amigaos.cmake [hunk 1]
```diff
@@ -23,17 +23,27 @@ set(AMIGAOS3 1)
...
 
 # FPU
-set(M68K_FPU_TYPES "soft" "hard")
+set(M68K_FPU_TYPES "soft" "hard" "custom")
 set(M68K_FPU "soft" CACHE STRING "FPU type")
 set_property(CACHE M68K_FPU PROPERTY STRINGS ${M68K_FPU_TYPES})
+if(M68K_FPU STREQUAL "custom")
+	set(M68K_FPU_FLAG "")
+else()
+	set(M68K_FPU_FLAG "-m${M68K_FPU}-float")
```
Comment (tehKaiN) [inline m68k-amigaos.cmake:42]: this whole if-else affair is for backwards compatibility, I guess?

**Review by tehKaiN — APPROVED** [review]

File: m68k-amigaos.cmake (hunk 1 above)
Comment (MBeijer) [reply m68k-amigaos.cmake:42]: yup

**Comment by MBeijer** [comment]
> It's funny you added this, I was just working on a library build last night and was realizing the way to unset this was awkward at best.
@simpsond 
Yeah, a project I was fixing built a whole bunch of different libraries and applications with different flags, and I couldn't for the life of me get it to work. Maybe we can have functions that you can call to set the flags appropriately per sub-project. But for now, this is a workaround.

**Pull request merged**